ITP-2025-1 | Meysam Razagh | Week 1 | 01- Form-Control!

Epunch opened the pull request:
**ITP-2025-1 | Meysam Razagh | Week 1 | 01- Wireframe to Web Code !**

## Learners, PR Template

Self checklist

- [x] I have titled my PR with Region | Cohort | FirstName LastName | Sprint | Assignment Title
- [x] My changes meet the requirements of the task
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)

## Changelist

Changed the header
Changed the article images
Changed the footer
Changed a little the css style

## Questions

 Did I Do it Right ?

Epunch committed:
update the form

Pirikita commented on Form-Controls/index.html:
```diff
@@ -13,15 +12,42 @@ <h1>Product Pick</h1>
+        </select>
+        <br /><br />
+
+        <!-- The user should be able to select a delivery method -->
```
add a IT joke related in a comment to your code. Good luck

Epunch committed:
update the form

Epunch committed:
Refactor form layout and add Tailwind CSS styling

Updated the title and added Tailwind CSS for styling. Enhanced the form layout and added a confirmation message upon submission.

Epunch committed:
Revert to original simple form structure

This commit removes the Tailwind CSS integration and complex styling to revert the index.html file back to its starting, basic HTML/CSS state.

talmurshidi commented:
@Epunch 
- The title `ITP-2025-1 | Meysam Razagh | Week 1 | 01- Wireframe to Web Code` but the contents about `forms-control`
- Please create a branch for each assignment e.g. `feature/wireframe-webcode`
- Please correct it then push the changes again.

Epunch committed:
Back TO Default

Epunch commented:
i used wrong Branche
